src/Pim/Bundle/CatalogBundle/Elasticsearch/Sorter/CompletenessSorter.php

```diff
@@ -9,7 +9,7 @@
  * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
  * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
  */
-class CompletenessSorter extends BaseFieldSorter
+class CompletenessSorter extends IdentifierSorter
```

Comment:
That is very strange to see that. No functional sense.
 I would prefer your Identifier sorter extend the BaseFieldSorter and just add what he need.